src/govuk/helpers/_links.scss

```diff
@@ -236,6 +284,49 @@
   }
 }
 
+/// Inverse style link mixin
+///
+/// Overrides the colour of links to work on a dark background.
+///
+/// If you use this mixin in a component you must also include the
+/// govuk-link-common mixin in order to get the correct focus and hover states.
+///
+/// @example scss
+///   .govuk-component__link {
+///     @include govuk-link-common;
+///     @include govuk-link-style-inverse;
+///   }
+///
+/// @access public
+
+@mixin govuk-link-style-inverse {
+  // Override link colour to use text colour
```

Comment:
I'm not sure what this comment means, as we're setting the link colour to white?

Reply:
Copy and paste fail! Great spot 👀

Reply:
I've removed this comment from this mixin and the text-colour mixin because I don't think it's particularly useful.